### ISSUE TITLE
Change gray form text to black

### ISSUE THREAD

**Issue by mi-wood** [RefugeRestrooms/refugerestrooms#365]
For some reason form text is gray and it's kind of hard to read in contrast on white

**Comment by ShayShayBeekay** [comment]
Can this issue be set to closed if resolved by 370?

**Comment by Hkly** [comment]
Whoops sorry, my bad, must have missed it when I reorganized the stylesheets.

**Comment by DeeDeeG** [comment]
Hi all.

On the latest develop branch, typed-in or filled-in form fields now have black text. (*placeholder text* is still gray, which I think it's supposed to be.)

![form text is black now](https://user-images.githubusercontent.com/20157115/30669670-2ba1b488-9e2d-11e7-85da-2697cb700221.png)


(Also It's not deployed to https://refugerestrooms.org yet, so form text is still gray there even when filled in.)

So basically I think this is fixed, at least on the develop branch.
